src/Moq/Obsolete/MockFactory.cs

```diff
@@ -293,6 +293,30 @@ public Mock<T> Create<T>(MockBehavior behavior, params object[] args)
 			return CreateMock<T>(behavior, args);
 		}
 
+		/// <summary>
+		/// Creates an instance of the mock using the given constructor call including its
+		/// argument values and with a specific <see cref="MockBehavior"/> behavior.
+		/// </summary>
+		/// <typeparam name="T">Type to mock.</typeparam>
+		/// <param name="newExpression">Lambda expression that creates an instance of <typeparamref name="T"/> T.</param>
+		/// <param name="behavior">Behavior of the mock.</param>
+		/// <returns>A new <see cref="Mock{T}"/>.</returns>
+		/// <example ignore="true">
+		/// <code>
+		/// var factory = new MockFactory(MockBehavior.Default);
+		/// 
+		/// var mock = factory.Create&lt;MyClass&gt;(() => new MyClass("Foo", 25, true));
+		/// // use mock on tests
+		/// 
+		/// factory.Verify();
+		/// </code>
+		/// </example>
+		public Mock<T> Create<T>(Expression<Func<T>> newExpression, MockBehavior behavior = MockBehavior.Strict)
```

Comment:
* `behavior` must default to `MockBehavior.Default` (reason given in a previous review comment).

* Moq's public API in general does not use optional parameters, but instead sets default values through different overloads and constructor / method chaining. It would be great if you could follow that pattern (for consistency).